frameworks/elastic/universe/package.json

```diff
@@ -6,7 +6,7 @@
   "name": "elastic",
   "version": "{{package-version}}",
   "maintainer": "support@mesosphere.io",
-  "description": "Elasticsearch 5, and optionally X-Pack",
```

Comment:
As X-pack is coming inbuld with es 6.3.1

Reply:
Yes, it is bundled with ES 6, but the users still have to opt-in to use it?